composer.json

```diff
@@ -144,7 +144,7 @@
         "monolog/monolog": "^3.0",
         "nikic/php-parser": "^4.18|^5.0",
         "nyholm/psr7": "^1.0",
-        "pda/pheanstalk": "^4.0",
+        "pda/pheanstalk": "^5.1|^7.0",
```

Comment:
These are the versions have the `disconnect` method, which is needed for the reconnect mechanism.